presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/PlanSanityChecker.java

```diff
@@ -42,6 +42,14 @@ public PlanSanityChecker(boolean forceSingleNode)
                         new NoSubqueryExpressionLeftChecker(),
                         new NoIdentifierLeftChecker(),
                         new VerifyOnlyOneOutputNode())
+                .putAll(
+                        Stage.FRAGMENT,
```

Comment:
`FRAGMENTED`? Looks like it should be an adjective.

Reply:
Ditto